validator/node/node.go

```diff
@@ -168,7 +168,7 @@ func (s *ValidatorClient) initializeFromCLI(cliCtx *cli.Context) error {
 			if err != nil {
 				return errors.Wrap(err, "could not generate interop keys")
 			}
-			accountsDir = cliCtx.String(flags.KeystorePathFlag.Name)
+			dbDir = cliCtx.String(flags.KeystorePathFlag.Name)
```

Comment:
this is still the accounts directory. I dont think we should assign this as the db dir, this will confuse any new reader.

Reply:
same db dir as here
`dbDir = cliCtx.String(cmd.DataDirFlag.Name)`